packaging/linux/Arch/PKGBUILD

```diff
@@ -69,43 +114,74 @@ prepare() {
 build() {
     export BRANCH="@GITHUB_BRANCH@"
     export BUILD_VERSION="@BUILD_VERSION@"
-    export COMMIT="@GITHUB_COMMIT@"
+    export COMMIT="${_commit}"
 
     export CC="gcc-${_gcc_version}"
     export CXX="g++-${_gcc_version}"
 
     export CFLAGS="${CFLAGS/-Werror=format-security/}"
     export CXXFLAGS="${CXXFLAGS/-Werror=format-security/}"
 
-    cmake \
-        -S "$pkgname" \
-        -B build \
-        -Wno-dev \
-        -D BUILD_DOCS=OFF \
-        -D BUILD_WERROR=ON \
-        -D CMAKE_INSTALL_PREFIX=/usr \
-        -D SUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine \
-        -D SUNSHINE_ASSETS_DIR="share/sunshine" \
-        -D SUNSHINE_PUBLISHER_NAME='LizardByte' \
-        -D SUNSHINE_PUBLISHER_WEBSITE='https://app.lizardbyte.dev' \
-        -D SUNSHINE_PUBLISHER_ISSUE_URL='https://app.lizardbyte.dev/support'
+    export CUDA_PATH=/opt/cuda
+    export NVCC_CCBIN="/usr/bin/g++-$_cuda_gcc_version"
+
+    export MAKEFLAGS="-j$(nproc)"
+
+    local _cmake_options=(
+      -S "$pkgname"
+      -B build
+      -Wno-dev
+      -D BUILD_DOCS=OFF
+      -D BUILD_WERROR=ON
+      -D CMAKE_INSTALL_PREFIX=/usr
+      -D SUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine
+      -D SUNSHINE_ASSETS_DIR="share/sunshine"
+      -D SUNSHINE_PUBLISHER_NAME='LizardByte'
+      -D SUNSHINE_PUBLISHER_WEBSITE='https://app.lizardbyte.dev'
+      -D SUNSHINE_PUBLISHER_ISSUE_URL='https://app.lizardbyte.dev/support'
+      -D SUNSHINE_ENABLE_CUDA=ON
+      -D SUNSHINE_ENABLE_DRM=ON
+      -D SUNSHINE_ENABLE_TRAY=ON
+      -D SUNSHINE_ENABLE_VAAPI=ON
+      -D SUNSHINE_ENABLE_WAYLAND=ON
+      -D SUNSHINE_ENABLE_X11=ON
```

Comment:
```suggestion
```

These are all ON by default. https://github.com/LizardByte/Sunshine/blob/25e1d415d2379c827814d7bc806efa68d4619757/cmake/prep/options.cmake#L56-L66

Reply:
Oh, probably legacy from before we failed if the required components were missing.